src/keboola_mcp_server/tools/sql.py

```diff
@@ -14,13 +14,17 @@
 LOG = logging.getLogger(__name__)
 
 SQL_TOOLS_TAG = 'sql'
+MAX_ROWS = 1_000
+MAX_CHARS = 50_000
 
 
 class QueryDataOutput(BaseModel):
     """Output model for SQL query results."""
 
     query_name: str = Field(description='The name of the executed query')
     csv_data: str = Field(description='The retrieved data in CSV format')
+    csv_data_rows: int | None = Field(default=None, description='Number of rows fetched in the "csv_data" field.')
+    total_query_rows: int | None = Field(default=None, description='Total number of rows selected by the SQL query.')
```

Comment:
I am still not sure from this description what exactly is the difference between the two fields. I thought that `csv_data_rows` could be lower than `total_query_rows` if there was truncation, but currently it looks like we don't do truncation.